lib/libxml-to-js.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
-var lodash = require('lodash');
+var lodash = {
+  isEmpty: require('lodash.isempty'),
```

Comment:
Used to do something like this, but then became extremely lazy. Thanks for the cleanup.